src/popup/components/TransactionOverview.vue

```diff
@@ -78,8 +79,7 @@ export default defineComponent({
       getOwnershipAddress,
       innerTx,
     } = useTransactionData({
-      transaction: props.transaction,
-      externalAddress: props.transaction?.transactionOwner,
```

Comment:
Are you sure that `externalAddress` ~ `transactionCustomOwner` should not be set here?